Makefile

```diff
@@ -13,7 +13,11 @@ BINDIR ?= $(PREFIX)/bin
 MANDIR ?= $(PREFIX)/share/man
 MAN1DIR ?= $(MANDIR)/man1
 SYSCONFDIR ?= $(PREFIX)/etc
```

Comment:
Also change this back to just "/etc" ? It's a little confusing if the config file earlyoom.default is not stored in /etc.

Reply:
If earlyoom.default is meant to be read by earlyoom main program, then this file and the main program should point to the same PREFIX, e.g. /usr/local.

The systemd service is meant to be read by the external systemd, therefore it will be put in another location not necessarily equal to the PREFIX of earlyoom - usually /etc/systemd/system/.

Is there anything wrong with the above?

Reply:
Yes, earlyoom.default is read by systemd, here: https://github.com/rfjakob/earlyoom/blob/master/earlyoom.service.in#L6

Reply:
But the Makefile deals with it already:

https://github.com/rfjakob/earlyoom/blob/04e93dea4dc74d2d4caaf5d7d48706cdebeb9586/Makefile#L51

But let's do as you suggested.

Reply:
Yeah it will work, but I find it confusing to have the config file for /etc/systemd/system/earlyoom.service in /usr/local . Thanks!